proposals/compatibility-mode.md

```diff
@@ -250,11 +250,19 @@ When supported, `float16-renderable` allows the `RENDER_ATTACHMENT` usage on tex
 When supported, `float32-renderable` allows the `RENDER_ATTACHMENT` usage on textures with format `"r32float"`, `"rg32float"`, and `"rgba32float"`.
 
 Without support, an error will occur at texture creation time as described in section 6.1.3.
+**Note these features do _not_ allow creating multisample textures with these formats.**
 
 Support for both features is mandatory in core WebGPU.
 
 **Justification**: OpenGL ES 3.1 does not require the relevant f16- or f32-based texture formats (`R16F`, `RG16F`, `RGBA16F`, `R32F`, `RG32F`, and `RGBA32F`) to be color-renderable. While there exist OpenGL ES extensions to enable renderability (`GL_EXT_COLOR_BUFFER_HALF_FLOAT` and `GL_EXT_COLOR_BUFFER_FLOAT`), there are a significant number of devices which lack support for these extensions.
 
+### 23. Disallow multisampled `*16float` and `*32float` textures
+
+Creating a texture with format `*16float` or `*32float` and `sampleCount > 1` produces a validation error.
```

Comment:
Just to make sure this is aware and discussed: 
I don't know if it worth seperating r16float and rg16float into a seperate case.

Reply:
Oh, I missed that there was a difference. Could you summarize it on #5049?

Reply:
Maybe a table of (all 6 formats) x (WebGPU Core vs. GLES3.1), each being "required" or "optional"